api/pkg/server/handlers.go

```diff
@@ -194,35 +194,6 @@ func (apiServer *HelixAPIServer) createSession(res http.ResponseWriter, req *htt
 		return nil, err
 	}
 
-	helixModel := req.FormValue("helixModel")
```

Comment:
are you sure this doesn't break the frontend?

Reply:
ah I see the refactor now, sorry

Reply:
Yeah this is an odd one - we don't even use the `POST /api/v1/sessions` route any more - we post to the `/chat` and `/learn` endpoints now

So - this old code is basically dead - and yet I refactor the function that filters the model name because it stinks that it's in two places but I'm not quite mentally ready to be deleting big chunks of code in this PR

So we get this strange middle ground that actually doesn't do anything :rofl:

Reply:
OK this is not true - the preview for tools and apps both still use the old `POST /api/v1/sessions` route - we should tidy these both up to use the new `/chat` route